lib/instrumentation/span.js

```diff
@@ -16,15 +16,16 @@ module.exports = Span
 util.inherits(Span, GenericSpan)
 
 function Span (transaction, name, type, opts) {
+  var childOf = transaction._agent._instrumentation.activeSpan || transaction
```

Comment:
Those are two very different things: `childOf` is just expressing the parent -> child relationship of spans while `activeSpan` maintains the async call graph to know the causality of things so it can link things back to the correct transaction, since the JS part of Node.js is a single-thread event loop.

Reply:
Hmmm yes, I suppose technically the parent isn't of the same transaction.